### ISSUE TITLE
Sign in fail

### ISSUE THREAD

**Issue by jiahaoChen** [Helixform/CodeCursor#35]
Hi, 
thanks for your development, the plugin is very convenient,
I encounter a problem, when press login, my chrome open the url
https://www.cursor.so/loginDeepPage
```
You may now proceed back to Cursor
If you have any trouble, please contact admin@cursor.so
```
and then I go back to VSCode, it still doesn't login,
any suggestion?

**Comment by zhangzhuang15** [comment]
I meet this problem too, do you solve it ?

**Comment by jiahaoChen** [comment]
not yet...

**Comment by zhangzhuang15** [comment]
你好 ，您发给我的邮件我已经收到。

**Comment by semal** [comment]
same problem, maybe forbid by the cursor?

**Comment by ktiays** [comment]
> Hi, thanks for your development, the plugin is very convenient, I encounter a problem, when press login, my chrome open the url https://www.cursor.so/loginDeepPage
> 
> ```
> You may now proceed back to Cursor
> If you have any trouble, please contact admin@cursor.so
> ```
> 
> and then I go back to VSCode, it still doesn't login, any suggestion?

Thank you for your feedback. 
Does the "Waiting for sign in / sign up..." dialog box still exist and not disappear automatically after returning to VSCode from Chrome?

**Comment by Tianhezhiwai** [comment]
same problem

**Comment by semal** [comment]
> 

dialog box disappear.

**Comment by JackSlowFucck** [comment]
me too.What to do with this problem, Boss

**Comment by Ericsongyl** [comment]
是啊，最新版的插件的，还是登录不了

**Comment by Cooper1023** [comment]
There is still problem.

**Comment by cyDione** [comment]
+1

**Comment by ivanvmoreno** [comment]
Facing same problem as of today - not yet solved

**Comment by zhangzhuang15** [comment]
May be a problem from the upstream.I cannot login Cursor App and even cannot visit the Cursor official website today.For me, I intend to wait for the reply from the Cursor repo in Github. @ivanvmoreno

**Comment by zhangzhuang15** [comment]
the [reason](https://github.com/getcursor/cursor/issues/613) @ivanvmoreno

**Comment by Elylicery** [comment]
same problem

**Comment by zgldh** [comment]
Got banned by Cursor maybe

**Comment by zhangzhuang15** [comment]
Unfortunately guys, I find that Cursor App brings big changes to  the api which is related with our login action and chat action.It blocks us using this code cursor plugin.

**Comment by unixzii** [comment]
This issue has been fixed in PR #43.
